### ISSUE TITLE
Error module on init

### ISSUE THREAD

**Issue by Giovancruz** [danguilherme/spotivy#31]
After installed, no commands work. The following error returned when i tryed to run spotify:

```
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'm3u8stream/lib/parse-time'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/gcruz/.nvm/versions/node/v10.15.3/lib/node_modules/spotivy/node_modules/ytdl-core/lib/info-extras.js:5:21)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/gcruz/.nvm/versions/node/v10.15.3/lib/node_modules/spotivy/node_modules/ytdl-core/lib/info.js:6:21)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
```
Wich is the requirements? Can you help me?

Node version 10.15.3
npm version 6.4.1

**Comment by danguilherme** [comment]
Thanks for the issue! Which command exactly did you run?

Also, could you post the result of running `spotivy --version`, and your command with the `--verbose` flag? (Make sure to remove any keys before posting.)

**Comment by Giovancruz** [comment]
Hi danguilherme,

Any command. spotivy, help, init with flag or without. All return the same error.

**Comment by Giovancruz** [comment]
Hi, @danguilherme, can you reproduce the error?

**Comment by danguilherme** [comment]
Yes, I can. Sorry for the delay!
The issue is with a dependency: https://github.com/fent/node-ytdl-core/issues/519

Updating the package to the latest version solves the issue, but the feature to download tracks as audio is broken.

I will take some time to fix it as soon as possible, and test the CLI to make sure no regression is found.

**Comment by Giovancruz** [comment]
Oh ok. Thankyou!

**Comment by danguilherme** [comment]
Fixed in v0.4.8.

@Giovancruz please test it if it works for you.
